widget/embedded/src/containers/Wallets/Wallets.tsx

```diff
@@ -102,12 +102,6 @@ function Main(props: PropsWithChildren<PropTypes>) {
   };
   const isActiveTab = useUiStore.use.isActiveTab();
 
-  useEffect(() => {
-    if (accounts.length) {
-      connectWallet(accounts, tokens);
-    }
-  }, [accounts, tokens]);
-
```

Comment:
I'm not sure about this change? What was the initial reason for this code?

Reply:
It was my change in one of my commits to ensure the switch accounts API call happens once. With the cleanup listener names fixed, we can revert it.